### ISSUE TITLE
Add the rest of the pages to translate

### ISSUE THREAD

**Issue by lokalisoija** [rocky-linux/rockylinux.org-gatsby#315]
Add the rest of the pages, such as https://rockylinux.org/keys, https://rockylinux.org/organizational-structure and 404 page to be translated into Crowdin.

**Comment by FoggyMtnDrifter** [comment]
Some pages are excluded from translations for legal reasons (we don't want any legal terms to get lost in translation), however I plan to include some of the non-legal pages soon.

**Comment by lokalisoija** [comment]
> Some pages are excluded from translations for legal reasons (we don't want any legal terms to get lost in translation), however I plan to include some of the non-legal pages soon.

Yep, legal pages and blog posts are not worth translating. However, other texts should be translated for consistency.

Could you open [this issue](https://github.com/rocky-linux/rockylinux.org/issues/313#issuecomment-999740533) again?

**Comment by lokalisoija** [comment]
> > Some pages are excluded from translations for legal reasons (we don't want any legal terms to get lost in translation), however I plan to include some of the non-legal pages soon.
> 
> Yep, legal pages and blog posts are not worth translating. However, other texts should be translated for consistency.
> 
> Could you open [this issue](https://github.com/rocky-linux/rockylinux.org/issues/313#issuecomment-999740533) again?

@ressonix

**Comment by FoggyMtnDrifter** [comment]
Hey there! I will not be re-opening that issue. 

Instead, after some consideration, I have decided not to automatically redirect the website based on the browser language, so that issue won't be relevant in this case.

**Comment by lokalisoija** [comment]
Okay, are there any plans to start using any other way(like based on cookie)?
